iphone/iphone/Assets.xcassets/AppIcon.appiconset/Contents.json

```diff
@@ -103,10 +103,34 @@
       "idiom" : "ios-marketing",
       "filename" : "appicon-1024.png",
       "scale" : "1x"
+    },
+    {
+         "appearances" : [
+           {
+             "appearance" : "luminosity",
+             "value" : "dark"
+           }
+         ],
+         "filename" : "DefaultIcon-Dark.png",
```

Comment:
What happens if this one does not exist? For example, some app icons adapt automatically to dark mode, so they don't need to have a manual version

Reply:
I will double check what happens if the file does not exist, but I believe when I was testing originally if the dark mode icon file is not there, it will just use the regular default icon and the system creates its own dark mode icon from it.